README.md

```diff
@@ -11,7 +11,8 @@ Touchstone currently supports comparison of the following docs:
 |      Uperf     |  Elasticsearch   |    Ripsaw     |
 |      YCSB      |  Elasticsearch   |    Ripsaw     |
 |      Pgbench   |  Elasticsearch   |    Ripsaw     |
-|      Vegeta    |  Elasticsearch   |    Ripsaw
+|      Vegeta    |  Elasticsearch   |    Ripsaw     |
+|        -       |  Prometheus      |       -       |
```

Comment:
I find this to be confusing. Maybe just have a separate call out to Prometheus support?